### PR TITLE
Switch to controller-runtime 0.7.0-alpha.0

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Using the new controller runtime for testing purposes.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: internal/pkg/controllers/profile/profile.go [hunk 1]
```diff
@@ -121,12 +121,9 @@ type Reconciler struct {
 }
 
 // Reconcile reconciles a SeccompProfile or a ConfigMap representing a seccomp profile.
-func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
+func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
 	logger := r.log.WithValues("profile", req.Name, "namespace", req.Namespace)
 
-	ctx, cancel := context.WithTimeout(context.Background(), reconcileTimeout)
```
Comment (hasheddan) [inline internal/pkg/controllers/profile/profile.go:127]: Hmm don't we want this context to be per-reconcile?

Comment (hasheddan) [reply internal/pkg/controllers/profile/profile.go:127]: @saschagrunert we are not actively working on it (updated fairly recently), but we would love for you to contribute it :)

**Comment by saschagrunert** [comment]
We'll switch to v0.8.0 directly once the crossplane PR got merged and released.
/close

**Pull request closed without merging**